### PR TITLE
Add Chinese (Simplified) & Chinese (Traditional)

### PR DESCRIPTION
**Pull request by Taylcd**

### CODE REVIEW

**Comment by SOF3** [comment]
This is not Chinese. This is simplified Chinese.

**Comment by SOF3** [comment]
Note that `zh` is a two-letter code from ISO 639-1, and `zho` is a three-letter code from ISO 639-2. In fact they represent the same language. It may be inadvisable to use a code from ISO 639-2, inconsistent with other languages, as an identifier for traditional Chinese. Instead, using another tag (e.g. zh-HK, zh-Hant, etc.) might be more meaningful.

Reference: https://www.loc.gov/standards/iso639-2/php/code_list.php https://www.w3schools.com/tags/ref_language_codes.asp

**Comment by Sandertv** [comment]
@SOF3 does this seem good to you?

**Comment by SOF3** [comment]
To be politically neutral, I suggest using `zh-hans` for simplified Chinese and don't have a default `zh`.

File: src/BlockHorizons/BlockSniper/Loader.php [hunk 1]
```diff
@@ -38,7 +38,9 @@ class Loader extends PluginBase {
 		"nl",
 		"fr",
 		"ko",
-		"ja"
+		"ja",
+        "zh-hans",
+        "zh-hant"
```
Comment (SOF3) [inline src/BlockHorizons/BlockSniper/Loader.php]: Actually wrong indentation here

Comment (Sandertv) [reply src/BlockHorizons/BlockSniper/Loader.php]: That's fine, I'll fix it later.

**Pull request merged**